lib/resources/r4/observation.yaml

```diff
@@ -30,22 +87,216 @@ fields:
+  example: |
+    {
+      "code": {
+        "coding": [
+          {
+            "system": 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/72',
+            "code": '4154126',
+            "display": 'Height/Length Measured',
+            "userSelected": true
+          }
+        ],
+        "text": 'Height/Length Measured'
+      }
+    }
+  example2: |
+    {
+      "code": {
+        "coding": [
+          {
+            "system": 'http://loinc.org',
+            "code": '8302-2'
+          },
+          {
+            "system": 'http://loinc.org',
+            "code": '3137-7'
+          }
+        ],
+        "text": 'Height/Length Measured'
+      }
+    }
+  note: |
+    <ul>
+      <li> Vital Signs currently support only proprietary codes </li>
+    </ul>  
+  url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.code
+
+- name: subject
+  required: 'Yes'
+  type: Reference
+  description: Indicates the patient who the observation record is associated with.
+  action: create
+  example: |
+    {
+      "subject": {
+        "reference": "Patient/631923"
+      }
+    }
+  note: |
+    <ul>
+      <li> Must only include references to Patient. </li>
+    </ul>
+  url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.subject
+
+- name: encounter
+  required: 'No'
+  type: Reference
+  description: Encounter during which the observation was first made.
+  action: create
+  example: |
+    {
+      "encounter": {
+        "reference": "Encounter/631923"
+      }
+    }
+  url: https://hl7.org/fhir/r4/observation-definitions.html#Observation.encounter
+
+- name: effective[x]
+  required: 'Yes'
+  type: dateTime
+  description: Date and time the observation was made.
+  action: create
+  example: |
+    {
+      "effectiveDateTime": "2020-07-07T20:37:00.000Z"
+    }
+  note: |
+    <ul>
+      <li> Only a type of dateTime is supported. </li>
+    </ul>
+  url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.effective_x_
+
+- name: issued
+  required: 'No'
+  type: instant
+  description: Instant the observation was made.
+  action: create
+  example: |
+    {
+      "issued": "2020-07-06T21:02:47.000Z"
+    }
+  url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.issued
+
+- name: performer
+  required: 'No'
+  type: List of Reference
+  description: Limited to "real" people rather than equipment.
+  action: create
+  example: |
+   {
+      "reference": 'Practitioner/12724064'
+    }
+  note: |
+    <ul>
+      <li> Must be a reference to Practitioner. </li>
+      <li> Vital Signs require performer and supports up to two performers. </li>
+    </ul>
+  url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.performer
+
+- name: performer.extension
+  required: 'No'
+  type: performerFunction
+  description:  Distinguishes the type of involvement of the performer in the event.
```

Comment:
is there extra space before `Distinguishes`?

Reply:
removed extra space : https://github.com/cerner/fhir.cerner.com/commit/a3c496078c9f8365f3c8e5b8e0e7d18a4d98b3ae#diff-53f5b7c6241dd914461705ddc9523c7eL201